addon/components/bs-navbar.js

```diff
@@ -7,38 +7,40 @@ import layout from '../templates/components/bs-navbar';
 
  ### Usage
 
- Used in combination with the following components:
+ Uses the following components by a contextual reference:
 
- - [Components.NavbarContent](Components.NavbarContent.html)
- - [Components.NavbarToggle](Components.NavbarToggle.html)
- - [Components.NavbarNav](Components.NavbarNav.html)
+ - [Components.NavbarHeader](Components.NavbarHeader.html
+ - [Components.NavbarContent](Components.NavbarContent.html
+ - [Components.NavbarToggle](Components.NavbarToggle.html
+ - [Components.NavbarBrand](Components.NavbarBrand.html
+ - [Components.NavbarNav](Components.NavbarNav.html
  - [Components.NavItem](Components.NavItem.html)
 
  ```hbs
- {{#bs-navbar}}
+ {{#bs-navbar as |navbar|}}
    <div class="navbar-header">
-     {{#bs-navbar-toggle}}
+     {{#navbar.toggle}}
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
-     {{/bs-navbar-toggle}}
+     {{/navbar.toggle}}
      <a class="navbar-brand" href="#">Brand</a>
    </div>
-   {{#bs-navbar-content}}
-     {{#bs-navbar-nav}}
-       {{#bs-nav-item}}{{#link-to "alert"}}Alert{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "button"}}Buttons{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "dropdown"}}Dropdown{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "forms"}}Forms{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "accordion"}}Accordion{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "collapse"}}Collapse{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "modal"}}Modals{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "progress"}}Progress bars{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "navs"}}Navs{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "navbars"}}Navbars{{/link-to}}{{/bs-nav-item}}
-     {{/bs-navbar-nav}}
-   {{/bs-navbar-content}}
+   {{#navbar.content}}
+     {{#navbar.nav}}
+       {{#navbar.nav-item}}{{#link-to "alert"}}Alert{{/link-to}}{{/navbar.nav-item}}
```

Comment:
Actually, since `e-b-navbar-nav` is derived from `e-b-nav` and templates aren't inherited, the nav/nav-item CC change won't inherit. I talked to some core folks, and components not inheriting templates is a deliberate choice. I think it would be bad to hack it in.

I debated whether to put it under the nav (e.g. `navbar.nav.item` or do it the way I did as `navbar.item`. This seemed simpler and sufficiently encapsulated, but it's a judgment call.